src/main/java/com/kainos/ea/backend/models/Training.java

```diff
@@ -0,0 +1,71 @@
+package com.kainos.ea.backend.models;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "Training")
+public class Training {
+
+    public enum TrainingType {
+        PROFESSIONAL_SKILLS,
+        TECHNICAL_SKILLS,
+        DEVELOPMENT_PROGRAMME
+    }
```

Comment:
Wouldn't it be better to get these from the db?

Reply:
Since there is no admin user story to edit the types of training, I don't think it would be worth the extra effort to do this